VERSION

```diff
@@ -1 +1 @@
-1.7.1-dev
+1.7.2
```

Comment:
Hi @patriciareinoso, I just noticed this but when you need to create a new release you only need to delete the `-dev` part of the version. That is the change should be as follows:
```diff
- 1.7.1-dev
+ 1.7.1
```